lib/Conways_Game.rb

```diff
@@ -0,0 +1,107 @@
+require 'matrix'
+
+class Conways_Game
```

Comment:
Classes are supposed to not have underscores, filename should be `conways_game.rb`